palette/src/alpha.rs

```diff
@@ -394,7 +396,7 @@ where
     C: fmt::LowerHex,
 {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        let size = f.width().unwrap_or(::core::mem::size_of::<T>() * 2);
+        let size = f.width().unwrap_or_else(|| ::core::mem::size_of::<T>() * 2);
```

Comment:
How come this was changed? I would expect `size_of` to be constant and not need a function wrapper.